projects/1.19/assets/chat-sounds/chatsounds/lang/zh_cn.json

```
@@ -0,0 +1,33 @@
+{
+    "text.autoconfig.chatsounds.title": "聊天音效",
+    "text.autoconfig.chatsounds.option.join": "上线信息",
+    "text.autoconfig.chatsounds.option.join.enabled": "启用",
+    "text.autoconfig.chatsounds.option.join.volume": "音量",
+    "text.autoconfig.chatsounds.option.join.pitch": "音高",
+    "text.autoconfig.chatsounds.option.join.sound": "音效 ID",
+    "text.autoconfig.chatsounds.option.leave": "离线信息",
```

```suggestion
    "text.autoconfig.chatsounds.option.leave": "退出消息",
```
原版`%s left the game`：`%s退出了游戏`